src/blmc_joint_module.cpp

```diff
@@ -298,6 +298,28 @@ bool BlmcJointModule::calibrate(double& angle_zero_to_index,
     return true;
 }
 
+void BlmcJointModule::init_homing_at_endstop(int joint_id,
```

Comment:
As this already finishes the whole procedure instantly here and doesn't directly depend on the end-stop, I would suggest, to rename this method to something like `home_at_current_position`.
Also the whole `homing_state_` is probably not needed.

Reply:
Yep, changed that, too.